<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTextDialogo</class>
 <widget class="QDialog" name="ImportTextDialogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add indata to the model</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>889</width>
     <height>761</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="PBAddInputFile">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Add input file</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="CBField">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <item>
         <property name="text">
          <string>--- Select field ---</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="CBCrop">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <item>
         <property name="text">
          <string>--- Select crop ---</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="PBHelp">
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>EPGS Coordinate system:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LEEPSG">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>4326</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>WGS 84 = 4326, RT90 2.5 gon V = 3021, UTM zone 32N = 32632, SWEREF99 TM = 3006 etc.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedKingdom"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="RBComma">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Comma</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RBSemi">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Semicolon</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RBTab">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Tab</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Separator</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RBOwnSep">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>0</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Other:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LEOwnSep">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="QTableWidget" name="TWtoParam">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="PBRemParam">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>&lt;--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="PBAddParam">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="LParams">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Parameters that could be analysed:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Columns in the file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="4">
       <widget class="QTableWidget" name="TWColumnNames">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QPushButton" name="PBContinue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="ComBEast">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="LMinYield">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Minimum yield:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>E:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEdit" name="LEMoveX">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="LMoveY">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Move (cc) forward:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>N:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="LEMinimumYield">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>10000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLineEdit" name="LEMoveY">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="LEMaximumYield">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>99999</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="ComBNorth">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="LMaxYield">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Maximum yield:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="LMoveX">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Move (cc) left:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="LMinYield_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Either select a column, fill in a static value or leave it</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="3">
       <widget class="QDialogButtonBox" name="button_box">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <layout class="QGridLayout" name="GLSpecific"/>
      </item>
      <item row="2" column="0">
       <widget class="QDateEdit" name="DE">
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="ComBDate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="RBUserTime">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Date is not in the file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Date format:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="RBDateOnly">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Date stored in 
following column</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="PBInsertDataIntoDB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Save data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="ComBDate_2">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>%Y-%m-%d %H:%M:%S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%Y-%m-%d %H:%M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%Y-%m-%d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%y%m%d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%Y-%m</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%m/%d/%Y %H:%M:%S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%m/%d/%y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%Y/%m/%d %H:%M:%S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%y/%m/%d</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="PBAbbreviations">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Abbreviations</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ImportTextDialogo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ImportTextDialogo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
